### ISSUE TITLE
x @@@C:cmd is wrong

### ISSUE THREAD

**Issue by ret2libc** [radareorg/radare2#16600]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Fedora 31 - x86_64
| File format of the file you reverse (mandatory)      | any
| Architecture/bits of the file (mandatory)            | x64
| r2 -v full output, **not truncated** (mandatory)     | radare2 4.4.0 27759 @ linux-x86-64 git.4.4.0-4-gf6528e171
commit: f6528e171e9caaef4d19e0a4bb506cd196e7d417 build: 2020-04-15__09:41:36

### Issue
The help message of `@@@C` says:
```
| x @@@C:cmd   comments matching
```
I think this is wrong, as `@@@C` probably does not need any `cmd` at the end. This is probably a wrong copy-paste from `@@@c:`. 

### Steps to reproduce
```
$ radare2 -
> Csa 10 @0xdeadbeef
> C
0xdeadbeef latin1[10] "\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff"
> pd 1 @@@C
```

### Actual behaviour
Nothing

### Expected behaviour
```
            0xdeadbeef     .string "\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff" ; len=10
```

**Comment by ret2libc** [comment]
@thestr4ng3r this is fixed, isn't it? AFAIR you fixed it not long ago. If not, please re-open.

**Comment by thestr4ng3r** [comment]
Yes, fixed in edf1be10d114e98b95aa1f6ddf7bbebf5279aa7e
